### PR TITLE
add minitest helper

### PR DESCRIPTION
**Pull request by ValentinoRusconi-EH**

### CODE REVIEW

**Review by mensfeld — APPROVED** [review]

File: lib/karafka/testing/minitest/proxy.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Karafka
+  module Testing
+    module Minitest
```
Comment (mensfeld) [inline lib/karafka/testing/minitest/proxy.rb:5]: I believe it is called `MiniTest` and not `Minitest`

**Review by mensfeld — CHANGES_REQUESTED** [review]
All good except the namespace as `MiniTest` aint `Minitest`

File: lib/karafka/testing/minitest/proxy.rb (hunk 1 above)
Comment (ValentinoRusconi-EH) [reply lib/karafka/testing/minitest/proxy.rb]: I adjusted

**Review by mensfeld — APPROVED** [review]

**Comment by nijikon** [comment]
@ValentinoRusconi-EH can you fix Coditsu errors?

**Comment by mensfeld** [comment]
@nijikon I asked @ValentinoRusconi-EH not to. I can just retarget that and fix myself not to keep it in the freezer longer. I'm interested about conceptual review here.

**Comment by mensfeld** [comment]
@nijikon please ping me when you are ok or not from the conceptual point. One more thing that I asked was NOT to generalize the rspec and minitest common code not to have inter-tooling dependency.

**Review by nijikon — APPROVED** [review]

**Comment by nijikon** [comment]
@mensfeld conceptually 👌

**Pull request merged**